web-app/src/app/screens/Feed/AssociatedFeeds.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { ContentBox } from '../../components/ContentBox';
+import {
+  Box,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableRow,
+  colors,
+} from '@mui/material';
+import { OpenInNewOutlined } from '@mui/icons-material';
+import {
+  type GTFSFeedType,
+  type AllFeedType,
+} from '../../services/feeds/utils';
+
+export interface AssociatedFeedsProps {
+  feeds: AllFeedType[] | undefined;
+}
+
+const renderAssociatedGTFSFeedRow = (
+  assocFeed: GTFSFeedType,
+): JSX.Element | undefined => {
+  if (assocFeed === undefined) {
+    return undefined;
+  }
+  return (
+    <TableRow key={assocFeed?.id}>
+      <TableCell>{assocFeed.feed_name}</TableCell>
+      <TableCell>
+        {assocFeed.latest_dataset?.downloaded_at !== undefined && (
+          <span style={{ display: 'flex' }}>
+            <a href={`/feeds/${assocFeed?.id}`} rel='noreferrer'>
+              Last updated on{' '}
+              {new Date(assocFeed.latest_dataset?.downloaded_at).toDateString()}
+            </a>
+            <OpenInNewOutlined />
```

Comment:
[suggestion] I would include the icon in the `a` tag, that or make the entire row clickable

Reply:
great idea -- done!